backlog_controller.hh

```diff
@@ -74,6 +74,8 @@ protected:
         return _static_shares > 0;
     }
 
+    static float slope(const control_point cp1, const control_point& cp2);
```

Comment:
let's return double since float precision sucks (and double is as native to the cpu as float so there's no reason to use float for calculations)

Reply:
float has ~23 bits of precision. Shares has 10 bits of precision, float isn't a problem.